sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/bigquery/BeamBigQuerySqlDialect.java

```diff
@@ -156,7 +156,11 @@ public void unparseCall(
         break;
       case OTHER_FUNCTION:
         String funName = call.getOperator().getName();
-        if (FUNCTIONS_USING_INTERVAL.contains(funName)) {
+        if (funName.equals("numeric_literal")) {
```

Comment:
Reverse the order here (`"numeric_literal".equals(funName)`) to avoid potential issues if funName is null. Also consider making "numeric_literal" a constant like the others.

Reply:
Done. Thanks.